internal/config/config_test.go

```diff
@@ -65,37 +65,26 @@ func TestReadDotEnv(t *testing.T) {
 		{"multiple variable", "SHIORI_HTTP_PORT=9999\nSHIORI_HTTP_SECRET_KEY=123123", map[string]string{"SHIORI_HTTP_PORT": "9999", "SHIORI_HTTP_SECRET_KEY": "123123"}},
 	} {
 		t.Run(testCase.name, func(t *testing.T) {
-			tmpDir, err := os.MkdirTemp("", "")
-			require.NoError(t, err)
-
+			tmpDir := t.TempDir() + "/local"
 			os.Chdir(tmpDir)
 
-			t.Cleanup(func() {
-				require.NoError(t, os.RemoveAll(tmpDir))
-			})
-
 			// Write the .env file in the temporary directory
 			handler, err := os.OpenFile(".env", os.O_CREATE|os.O_WRONLY, 0655)
 			require.NoError(t, err)
 			handler.Write([]byte(testCase.line + "\n"))
 			handler.Close()
 
 			e := readDotEnv(log)
+			os.RemoveAll(".env")
```

Comment:
I don't think youneed to remove the `.env` file since the test will remove the temp dir automatically

Reply:
So windows is ignoring the `os.Chdir` I guess? Mhmm

Reply:
Oh, `os.Chdir` returns an error and we are not checking it. Maybe we can get more information from that.

Reply:
I still think this shouldn't be required. If this test is not behaving properly in windows we should skip it in that system and go back to it once the upstream issue is resolved.